graylog2-server/src/main/java/org/graylog2/indexer/indices/Indices.java

```diff
@@ -428,4 +446,8 @@ public void refresh(String... indices) {
     public Map<String, Object> indexMapping(String index) {
         return indicesAdapter.getIndexMapping(index);
     }
+
+    public Map<String, Object> indexSettings(String index) {
+        return IndexSettingsHelper.unflatten(indicesAdapter.getFlattenIndexSettings(index));
```

Comment:
The reason was that we need to duplicate the code of `IndexSettingsHelper` because the implementations of the `IndicesAdapter` are in different modules. I don't know what is worse, having the duplication or how it is now. What do you think?

Reply:
I missed the problem with different modules. So let's leave it as is. I prefer that to duplication.